client/src/pages/Home.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+
+const Home = (props) => {
+    const [user, setUser] =  useState(localStorage.getItem('profile'));
+
+    let homeText;
+    if(user ==='' || user == null) {
+        homeText=(<div>You are not logged in</div>)
+    }
+    else{
+        homeText=(
+        <div>
+            <div>Hi {user} !</div>
+            <h1>Choose Options</h1>
+            <ul className="Home-options">
+                <li className="Home-options-elements">
+                    <button className="Home-options-elements-button">
+                        <input type="image" src="../../../Calendar.png" alt="Calendar" width="200" height="200" onClick={()=>{console.log("Clicked!")}} />
+                    </button>
+                </li>
+            </ul>
+
+            <ul className="Home-options">
+                <li className="Home-options-elements">
+                    <button className="Home-options-elements-button">
+                        <input type="image" src="../../../Check.png" alt="Calendar" width="200" height="200" onClick={()=>{console.log("Clicked!")}} />
+                    </button>
+                </li>
+            </ul>
```

Comment:
Is there a reason for wrapping buttons in unordered list element? We should also give a visible title to buttons.

Reply:
@Lee-Taeho said that he would modify it to make them side by side like what we had in the wireframe

Reply:
I have made a new version of buttons in the page-implementation branch, so you can delete this part if you want.